auth0/resource_auth0_custom_domain_verification_test.go

```diff
@@ -3,54 +3,36 @@ package auth0
 import (
 	"testing"
 
-	"github.com/alexkappa/terraform-provider-auth0/auth0/internal/digitalocean"
-	"github.com/alexkappa/terraform-provider-auth0/auth0/internal/random"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/terraform"
 )
 
 func TestAccCustomDomainVerification(t *testing.T) {
-
-	rand := random.String(6)
-
 	resource.Test(t, resource.TestCase{
 		Providers: map[string]terraform.ResourceProvider{
-			"auth0":        Provider(),
-			"digitalocean": digitalocean.Provider(),
+			"auth0": providerWithWiremock(),
```

Comment:
Checking my understanding here, this is the exact spot where we'd swap the real provider with the provider with mocked requests, when we adjust the tests going forward?

Reply:
Yep, that's correct:)